### ISSUE TITLE
After install could not start ember s again

### ISSUE THREAD

**Issue by sgasser** [imgix/ember-cli-imgix#3]
After install ember-cli-imgix via ```ember install ember-cli-imgix
``` I run ```ember s``` and I get following errors:

```
$ ember s
version: 1.13.12
Livereload server on http://localhost:49152
Serving on http://localhost:4200/
ENOENT, no such file or directory 'ember-app/tmp/concat_with_maps-input_base_path-TJpQil0w.tmp/0/bower_components/md5/build/md5.min.js'
Error: ENOENT, no such file or directory 'ember-app/tmp/concat_with_maps-input_base_path-TJpQil0w.tmp/0/bower_components/md5/build/md5.min.js'
    at Error (native)
    at Object.fs.statSync (fs.js:797:18)
    at ConcatWithMaps.keyForFile (ember-app/node_modules/ember-cli/node_modules/broccoli-sourcemap-concat/node_modules/broccoli-caching-writer/index.js:90:20)
    at Array.map (native)
    at ConcatWithMaps.CachingWriter._conditionalBuild (ember-app/node_modules/ember-cli/node_modules/broccoli-sourcemap-concat/node_modules/broccoli-caching-writer/index.js:112:65)
    at ember-app/node_modules/ember-cli/node_modules/broccoli-sourcemap-concat/node_modules/broccoli-caching-writer/node_modules/broccoli-plugin/read_compat.js:61:34
    at lib$rsvp$$internal$$tryCatch (ember-app/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:493:16)
    at lib$rsvp$$internal$$invokeCallback (ember-app/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:505:17)
    at lib$rsvp$$internal$$publish (ember-app/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:476:11)
    at lib$rsvp$asap$$flush (ember-app/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:1198:9)

```

**Comment by sgasser** [comment]
I think there is missing the md5 package in bower.json

**Comment by kellysutton** [comment]
Hi @sgasser, the md5 package is a subdependency. It should be pulled in by imgix-core-js: https://github.com/imgix/imgix-core-js. A `bower install` should fix the issue. Can you try running that and report back?

**Comment by sgasser** [comment]
Hi @kellysutton,

no difference with ```bower install```
In bower_components all bower components for ember-cli-imgix are missing...

**Comment by kellysutton** [comment]
Interesting. I know the last version of ember-cli that this was confirmed to work with was 1.13.8. There might be some transition work required in upgrading it to be compatible with 1.13.12.

**Comment by paulstraw** [comment]
I'm taking a peek at this. Interestingly, I was able to reproduce this issue with both ember-cli `1.13.8` _and_ `1.13.13`. Will keep digging.

**Comment by paulstraw** [comment]
Hey @sgasser, this should be fixed up in `0.1.2`. Please give it a shot and let me know!

**Comment by sgasser** [comment]
Fixed. Thx
